extension/src/plots/webview/messages.ts

```diff
@@ -79,7 +78,10 @@ export class WebviewMessages {
   public handleMessageFromWebview(message: MessageFromWebview) {
     switch (message.type) {
       case MessageFromWebviewType.ADD_CUSTOM_PLOT:
-        return commands.executeCommand(RegisteredCommands.PLOTS_CUSTOM_ADD)
+        return commands.executeCommand(
+          RegisteredCommands.PLOTS_CUSTOM_ADD,
+          this.dvcRoot
```

Comment:
[F] If there were multiple dvc projects in the workspace then clicking on the add custom plot button in the webview would have prompted the user for which project they wanted to run the add command against.